src/codeu/chat/client/core/View.java

```diff
@@ -136,4 +131,21 @@ public View(ConnectionSource source) {
 
     return messages;
   }
+
```

Comment:
Another style nit: Please stick with one blank line between methods for consistency.

Also, as elsewhere, please stick with the style of putting spaces before and after keywords like `if` and `else`.